src/spec/monasca_client.spec.js

```diff
@@ -1,14 +0,0 @@
-import {MonascaClient} from "../src/components/monasca_client"
-
-describe("Monasca Unit Tests", () => {
```

Comment:
It looks like this belongs in the previous commit where it was added (since we want to remove it)